lib/index.js

```diff
@@ -157,48 +157,59 @@ auth0GuardianJS.prototype.start = function start(callback) {
  */
 
 auth0GuardianJS.resume = function resume(options, transactionState, callback) {
-  var transactionTokenObject = jwtToken(transactionState.transactionToken);
-  var txId = transactionTokenObject.getDecoded().txid;
+  try {
+    var transactionTokenObject;
 
-  // create httpClient/socketClient
-  var httpClientInstance = object.get(options, 'dependencies.httpClient',
-    httpClient(transactionState.baseUrl, txId));
+    try {
+      transactionTokenObject = jwtToken(transactionState.transactionToken);
+    } catch (e) {
+      throw new errors.InvalidToken('Invalid transaction token');
```

Comment:
> Unfortunately there is no error name to check, it is just an instance of error, the only thing I can check is the message

That is awful :'(

I made a PR to add an InvalidTokenError, but it's ok if you don't want to wait for that.
https://github.com/auth0/jwt-decode/pull/48